server/db/migrations/20190528151100-create-user.js

```diff
@@ -45,6 +45,10 @@ export function up(queryInterface, Sequelize) {
     email_confirm_code: {
       type: Sequelize.STRING,
       allowNull: true
+    },
+    role: {
+      type: Sequelize.STRING,
```

Comment:
Inconsistency in the role type. The model has ENUM while migration has STRING

Reply:
An oversight, will fix that soon